crates/autopilot/src/database/auction_prices.rs

```diff
@@ -0,0 +1,28 @@
+use {
+    super::Postgres,
+    anyhow::Result,
+    primitive_types::H160,
+    shared::price_estimation::native::from_normalized_price,
+    std::collections::HashMap,
+};
+
+impl Postgres {
+    pub async fn fetch_latest_prices(&self) -> Result<HashMap<H160, f64>> {
```

Comment:
For DB functions that get used for the autopilot and the orderbook it's enough to implement it in the shared `database` crate.

Reply:
But we already have it: `database::auction_prices::fetch_latest_prices`. The sense of this function is to initialize separate metrics.

Reply:
The call is small enough that one could do it in the `main` functions. Overall I'm quite unhappy with our DB code because since it involves quite a bit of boiler plate. Could also make sense to implement this PR with the status quo (as you already did) and think more deeply about how things could be improved and have 1 big migration PR instead.
I'm open to suggestions.

Reply:
> it doesn't return raw structures, but already domain types

Each crate has its own domain. So, we have `database` modules in each crate with the same responsibility. Having a shared crate with mixed domain return types looks weird.